pom.xml

```diff
@@ -698,6 +698,70 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <version>2.44.5</version>
+                    <configuration>
+                        <skip>${spot.skip}</skip>
+                        <!--lineEndings>UNIX</lineEndings-->
+                        <!--formats>
+                            <format>
+                                <endWithNewline />
```

Comment:
does matter to me, only that its align on what in an random impl. detail.

You should define the styles then then apply what you like or remove.

Simply checkout and run the build to see diff.

@oowekyala @adangel